### ISSUE TITLE
"Could not find config" error when using -n, -a or -r in version 0.2.13

### ISSUE THREAD

**Issue by hyblm** [alanvardy/tod#139]
If you are experiencing this issue run the following in your terminal:
`sed -ie "s|$HOME/.tod.cfg|$XDG_CONFIG_HOME/tod.cfg|" $XDG_CONFIG_HOME/tod.cfg`

When moving an existing config file to the new location it keeps the old location in the JSON, which breaks functions that use it, namely `Config.save()`.

I can't currently think of a better way of solving this then just running above `sed` command in the code after moving the file. It works but doesn't seem ideal.

I am very sorry for introducing this problem to the code.

**Comment by alanvardy** [comment]
I just discovered this today too!

> When moving an existing config file to the new location it keeps the old location in the JSON, which breaks functions that use it, namely Config.save().

Thank you for identifying the issue, makes the fix a lot easier :) The bash command is really helpful too!

> I can't currently think of a better way of solving this then just running above sed command in the code after moving the file. It works but doesn't seem ideal.

I hacked on it for a bit and have a PR up, would you be interested in reviewing it?

> I am very sorry for introducing this problem to the code.

Don't be! I appreciate the contribution and I also missed the issue in code review. I also have enjoyed writing a little Rust again as I have been away from it for a few months.

**Comment by alanvardy** [comment]
https://github.com/alanvardy/tod/pull/140

Would like it if you could look it over, once it is merged I will release the fix :+1:

**Comment by hyblm** [comment]
Just looked through it and added a review, but I'm not sure if I did it correctly. This was my first open source contribution so I'm still unsure how things are done.

**Comment by alanvardy** [comment]
Yes, it is done correctly :+1: thank you for taking a look!

**Comment by alanvardy** [comment]
Thank you for the PR, this follow-up issue, and congrats on your first open source contribution @hyblm !

v0.2.14 with the fix is released to AUR and crates.io :+1:
